mne/viz/_figure.py

```diff
@@ -2295,6 +2074,88 @@ def _toggle_vline(self, visible):
         self.mne.vline_visible = visible
         self.canvas.draw_idle()
 
+    def close_all(self):
+        plt.close('all')
+
+    # workaround: plt.close() doesn't spawn close_event on Agg backend
+    # (check MPL github issue #18609; scheduled to be fixed by MPL 3.4)
+    def close_event(self, target=None):
+        """Force calling of the MPL figure close event."""
+        target = target or self
+        try:
+            target.canvas.close_event()
+        except ValueError:  # old mpl with Qt
+            pass  # pragma: no cover
+
+    def get_n_windows(self):
+        return len(plt.get_fignums())
```

Comment:
no. This will include any other MPL figure windows that the user happens to have open.

Reply:
Right, this shouldn't be public.